src/Lykke.AlgoStore.Api/Models/AlgoBackTestInstanceModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Lykke.AlgoStore.CSharp.AlgoTemplate.Models.Enumerators;
+
+namespace Lykke.AlgoStore.Api.Models
+{
+    public class AlgoBackTestInstanceModel
+    {
+        public string InstanceId { get; set; }
+        public string AlgoId { get; set; }
+        public string AlgoClientId { get; set; }
+        public string InstanceName { get; set; }        
+        public double BacktestTradingAssetBalance { get; set; }
```

Comment:
We don't need these properties in AlgoInstanceModel, because we don't store them in Algo Instance database, we need them only in Statistics